package.json

```diff
@@ -3,13 +3,31 @@
   "version": "0.15.5",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
+  "description": "A Web3 library for interacting with the ZkSync Layer 2 scaling solution.",
   "files": [
     "build/",
     "abi/",
     "src/",
     "typechain/"
   ],
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matter-labs/zksync-web3.git"
```

Comment:
`ethers.js` v5 is available on `ethers-v5` branch: https://github.com/zksync-sdk/zksync2-js/tree/ethers-v5